clients/drcachesim/tests/drmemtrace.threadsig.x64.tracedir/aux/info.textproto

```diff
@@ -0,0 +1,4 @@
+information {
+    peak_live_core_count: 4
+    phases: "may"
+}
```

Comment:
All windows CI failing with
```
Checking out the ref
  "C:\Program Files\Git\bin\git.exe" checkout --progress --force refs/remotes/pull/7430/merge
  Error: error: invalid path 'clients/drcachesim/tests/drmemtrace.threadsig.x64.tracedir/aux/info.textproto'
  Error: The process 'C:\Program Files\Git\bin\git.exe' failed with exit code 128
  ```
Not sure why `git.exe` complains.

Reply:
Try searching for the error string? E.g., https://stackoverflow.com/questions/63727594/github-git-checkout-returns-error-invalid-path-on-windows. Does it help?